jwst/regtest/test_niriss_image.py

```diff
@@ -44,6 +46,36 @@ def test_niriss_image_detector1(run_detector1, rtdata_module, fitsdiff_default_k
     _assert_is_same(rtdata_module, fitsdiff_default_kwargs, suffix)
 
 
+@pytest.mark.bigdata
+def test_niriss_tweakreg_no_sources(rtdata, fitsdiff_default_kwargs):
+    """Make sure tweakreg is skipped when sources are not found.
+    """
+    #rtdata = rtdata_module
+
+    cal_files = ["niriss/imaging/jw01537003001_02109_00001_nis_cal.fits",
+                 "niriss/imaging/jw01537003001_02109_00002_nis_cal.fits"]
+
+    for cal_file in cal_files:
+        rtdata.get_data(cal_file)
+
+    rtdata.input = "niriss/imaging/jw01537-o003_20240406t164421_image3_00004_asn.json"
+    rtdata.get_data("niriss/imaging/jw01537-o003_20240406t164421_image3_00004_asn.json")
```

Comment:
Instead of doing all the separate calls to `get_data` in order to load the asn file and the member cal files, you can instead just do 1 call to `rtdata.get_asn("niriss/imaging/jw01537-o003_20240406t164421_image3_00004_asn.json")`, which will fetch and load the asn file and all the members.

Reply:
Fixed. Thanks!